### ISSUE TITLE
Add support for server defined join address groups

### ISSUE THREAD

**Issue by Josh65-2201** [plan-player-analytics/Plan#3697]
### I would like to be able to..

I would like if the server config could have predefined groups that can have user control visibility in addition to user adding there own

```
join_groups:
    group_name: (show _ as a space in the UI)
        "url/ip"
```

### Is your feature request related to a problem? Please describe.

It very easy to lose groups and having some server defined ones give everyone viewing same data.

**Comment by AuroraLS3** [comment]
This has turned out to be surprisingly complicated since each server has its own join address list 🤔 I would assume server owner would like to configure all groups in one place on networks with multiple servers, but adding uuids to the config seems a bit cumbersome - and reusing one default group from the config for all servers doesn't seem to make sense

**Comment by ZepsiZola** [comment]
Yeah this kinda needs to be done. The Join Addresses page is basically useless if it doesn't remember what I'm trying to track. On top of that, it seems that I can't add more than one address into an "Address Group" in the page either.

**Comment by AuroraLS3** [comment]
That's how it currently works if you have login enabled - The address groups you make are stored in user preferences.

When http only they are stored in browser local storage.

This ticket is more about having default values for when you haven't created address groups

**Comment by ZepsiZola** [comment]
From what I can tell they wipe when I reload the page. I'm logged in as a user. I have reverse-proxied the webserver so maybe it thinks my instance is http-only? But I did follow the [guide](https://github.com/plan-player-analytics/Plan/wiki/SSL-Certificate-%28HTTPS%29-Set-Up#if-behind-a-proxy) on Authentication with reverse-proxy.

**Comment by ZepsiZola** [comment]
Should I make a bug-report somewhere else? I have a video I can submit of the bug taking place in case I might be doing something wrong.
